chibuezeayogu/React-Math-Skills-Game
# #160799053 Add button component

**What does the PR do?**
- Add `Button Component`

**Description of Task to be completed?**
- write component test
- add button component

**How should this be manually tested?**
- Clone the repository [here](https://github.com/chibuezeayogu/React-Math-Skills-Game.gitt)
-  cd into `React-Math-Skills-Game` and checkout to this branch `ft-add-button-component-160799053`
- run `npm start`
- run `npm test` and ensure all test are passing

**What are the relevant pivotal tracker stories?**
- [#160799053](https://www.pivotaltracker.com/story/show/160799053)

**Screenshots**
- N/A
